easybuild/easyconfigs/e/EveryBeam/EveryBeam-0.6.1-foss-2023b.eb

```
@@ -0,0 +1,70 @@
+easyblock = 'CMakeMake'
+
+name = 'EveryBeam'
+version = "0.6.1"
+
+homepage = 'https://everybeam.readthedocs.io/'
+description = """Library that provides the antenna response pattern for several instruments,
+such as LOFAR (and LOBES), SKA (OSKAR), MWA, JVLA, etc."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+sources = [
+    {
+        'filename': '%(name)s-v%(version)s.tar.gz',
+        # Repo uses git submodules, which are not included in the release tarballs.
+        # Thus, we let EasyBuild download directly from the git repository.
+        'git_config': {
+            'url': 'https://git.astron.nl/RD',
+            'repo_name': '%(name)s',
+            'tag': 'v%(version)s',
+            'clone_into': '%(name)s',
+            'recursive': True
+        }
+    },
+]
+patches = [
+    'EveryBeam-0.5.4_python_metadata.patch'
+]
+checksums = [
+    None,  # checksum for git clone source is non-deterministic
+    '0f66d50b1f7944ad0d64898a0e54ee8b20dc9fb3ec0d15b4e1da30bda88688ab'  # EveryBeam-0.5.4_python_metadata.patch
```

```suggestion
    '63c42e85d1b63cbf5887d10ec158be9b69e2c63cf3b209e9c1944f46c01810ac'  # EveryBeam-0.5.4_python_metadata.patch
```
(required for fixing a typo)